### ISSUE TITLE
issue on running examples

### ISSUE THREAD

**Issue by carlosPBC** [michaeldavie/env_canada#65]
Hi everyone,

In trying to run the example codes, I am not able to import as is stated: 
     from env_canada import ECHistoricalRange, get_historical_stations

I checked the source code, and it seems these methods have moved. Are these example still supposed to work as is?

**Comment by everettsp** [comment]
for some reason, the wildcard imports aren't grabbing everything - not sure whether it's due to a namespace conflict or something else. Can be resolved with:
```python
from env_canada.ec_historical import get_historical_stations
````
submitted PR with fix

**Comment by jeffmacdonald** [comment]
Ditto. Here's the error 

```
(env-canada) jeff.macdonald@mac-2207-jm env-canada % ./data-importer.py
Traceback (most recent call last):
  File "/Users/jeff.macdonald/env-canada/./data-importer.py", line 5, in <module>
    from env_canada import ECHistorical, get_historical_stations
ImportError: cannot import name 'get_historical_stations' from 'env_canada' (/Users/jeff.macdonald/env-canada/lib/python3.12/site-packages/env_canada/__init__.py)
```

**Comment by michaeldavie** [comment]
Fixed by PR #87
